### PR TITLE
Initial k8s scheduler content

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

File: applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Ensure that the --address argument is set to 127.0.0.1'
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml]: Change this to `Ensure that the --address argument is set` or `Ensure That The --address Argument Is Configured`

File: applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+severity: medium
+
+ocil_clause: '<tt>--address</tt> argument is not set to the IP address of the loopback device'
+
+ocil: |-
+    'Run the following command on the master node(s):
+     <pre>ps -ef | grep kube-apiserver</pre>
+
+    Verify that the <tt>--address</tt> argument is set to 127.0.0.1.'
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml]: should be `<tt>127.0.0.1</tt>`

Comment (redhatrises) [reply applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml]: or use `<pre/>`

File: applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+title: 'Ensure that the --profile argument is set to false'
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml]: Change this to `Ensure that the --profile argument is set` or `Ensure That The --profile Argument Is Configured`

File: applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+description: |-
+    'Profiling should be disabled if not needed. To disable profiling,
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml]: remove `'`

File: applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    not experiencing any bottlenecks and do not need the profiler for
+    troubleshooting purposes, it is recommended to turn it off to reduce the
+    potential attack surface.
+
+severity: low
+
+ocil_clause: 'profiling is not being used, and the <tt>--profiling</tt> argument is not disabled'
+
+ocil: |-
+    'Run the following command on the master node(s):
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml]: remove `'`

File: applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    for health and metrics information and is available without authentication
+    or encryption. As such it should only be bound to a localhost interface,
+    to minimize the cluster's attack surface.
+
+severity: medium
+
+ocil_clause: '<tt>--address</tt> argument is not set to the IP address of the loopback device'
+
+ocil: |-
+    'Run the following command on the master node(s):
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml]: remove `'`

File: applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Ensure that the --address argument is set to 127.0.0.1'
+
+description: |-
+    'To ensure the Scheduler service is bound to a secure loopback
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml]: remove `'`

File: applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    or encryption. As such it should only be bound to a localhost interface,
+    to minimize the cluster's attack surface.
+
+severity: medium
+
+ocil_clause: '<tt>--address</tt> argument is not set to the IP address of the loopback device'
+
+ocil: |-
+    'Run the following command on the master node(s):
+     <pre>ps -ef | grep kube-apiserver</pre>
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml]: Check all instances of `ps -ef` and change them to `$ ps -ef`

**Comment by shawndwells** [comment]
requested changes made - thanks!

**Comment by shawndwells** [comment]
@openscap-ci re-test this

File: applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    troubleshooting purposes, it is recommended to turn it off to reduce the
+    potential attack surface.
+
+severity: low
+
+ocil_clause: 'profiling is not being used, and the <tt>--profiling</tt> argument is not disabled'
+
+ocil: |-
+    Run the following command on the master node(s):
+    <pre>ps -ef | grep kube-apiserver</pre>
```
Comment (redhatrises) [inline applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml]: @shawndwells forgot a `$` here

**Comment by redhatrises** [comment]
ack.

**Comment by mpreisler** [comment]
The jenkins fail was real btw:
```
112/220 Test #113: validate-ssg-ocp3-ds.xml .................................................***Failed    0.22 sec
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/fedora/build/ssg-ocp3-ds.xml' line 3718: Element '{http://checklists.nist.gov/xccdf/1.2}select', attribute 'idref': 'dangling reference to address_argument!' is not a valid value of the atomic type 'xs:NCName'.
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/fedora/build/ssg-ocp3-ds.xml' line 3718: Element '{http://checklists.nist.gov/xccdf/1.2}select', attribute 'idref': Warning: No precomputed value available, the value was either invalid or something strange happend.
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/fedora/build/ssg-ocp3-ds.xml' line 3719: Element '{http://checklists.nist.gov/xccdf/1.2}select', attribute 'idref': 'dangling reference to profiling_argument!' is not a valid value of the atomic type 'xs:NCName'.
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/fedora/build/ssg-ocp3-ds.xml' line 3719: Element '{http://checklists.nist.gov/xccdf/1.2}select', attribute 'idref': Warning: No precomputed value available, the value was either invalid or something strange happend.
OpenSCAP Error: Invalid SCAP Source Datastream (1.2) content in /home/jenkins/workspace/scap-security-guide-pull-requests/label/fedora/build/ssg-ocp3-ds.xml. [oscap_source.c:342]

        Start 114: ansible-role-syntax-check-opensuse
```

Please make sure it passes jenkins before merging.

**Pull request merged**